--- built/lib/config.js	2017-05-09 17:44:18.000000000 +0000
+++ ../../package/built/lib/config.js	2018-03-13 00:14:50.903432307 +0000
@@ -3,6 +3,7 @@
 const os = require("os");
 const path = require("path");
 const cli_1 = require("./cli");
+const process = require("process");
 let logger = new cli_1.Logger('config');
 /**
  * The configuration for webdriver-manager
@@ -34,7 +35,7 @@
         return path.resolve(Config.dir, '..', Config.packageFile);
     }
     static getSeleniumDir() {
-        return path.resolve(Config.dir, '..', '..', 'selenium/');
+        return path.resolve(process.env.SELENIUM_DIR);
     }
     static getBaseDir() {
         return path.resolve(Config.dir, '..', '..');
@@ -91,4 +92,4 @@
 Config.httpsProxy_ = process.env.HTTPS_PROXY || process.env.https_proxy;
 Config.httpProxy_ = process.env.HTTP_PROXY || process.env.http_proxy;
 exports.Config = Config;
-//# sourceMappingURL=config.js.map
\ No newline at end of file
+//# sourceMappingURL=config.js.map
