--- bin/grunt	2016-05-13 18:06:14.385920115 +0000
+++ bin/grunt-new	2016-05-13 18:06:35.248920115 +0000
@@ -16,7 +16,6 @@
 
 
 var basedir = process.cwd();
-var gruntpath;
 
 // Do stuff based on CLI options.
 if ('completion' in options) {
@@ -29,17 +28,4 @@
   basedir = path.resolve(path.dirname(options.gruntfile));
 }
 
-try {
-  gruntpath = resolve('grunt', {basedir: basedir});
-} catch (ex) {
-  gruntpath = findup('lib/grunt.js');
-  // No grunt install found!
-  if (!gruntpath) {
-    if (options.version) { process.exit(); }
-    if (options.help) { info.help(); }
-    info.fatal('Unable to find local grunt.', 99);
-  }
-}
-
-// Everything looks good. Require local grunt and run it.
-require(gruntpath).cli();
+require('grunt').cli();
