--- /home/vagrant/package-original/lib/winston/transports/console.js	2018-05-09 19:50:23.972027118 +0000
+++ /home/vagrant/package/lib/winston/transports/console.js	2018-05-09 19:51:13.354445166 +0000
@@ -79,6 +79,7 @@
     logstash:    this.logstash,
     depth:       this.depth,
     formatter:   this.formatter,
+    showLevel:   this.showLevel,
     humanReadableUnhandledException: this.humanReadableUnhandledException
   });
 
